topic/autoscaling/README.md

```diff
@@ -21,29 +21,25 @@ about autoscaling. \<ADD LINK\>
 
 ## Usage
 
-Run a dedicated cluster in CrateDB Cloud:
-
-> <https://console.cratedb.cloud/>
+Run a dedicated cluster in [CrateDB Cloud](https://console.cratedb.cloud/).
 
 ### Install
 
 The script uses a couple of Python libraries. Make sure you have installed those:
+
 ```shell
-pip install datetime
```

Comment:
`datetime` is actually a different package: https://pypi.org/project/DateTime/. The actual `datetime` module comes with Python by default.